app/views/course/video/videos/_video.html.slim

```diff
@@ -10,6 +12,10 @@
   td.table-buttons
     div.btn-group.attempt
       = render partial: 'video_attempt_button', locals: { video: video }
+  td.table-buttons.hidden-xs
+    - if can?(:manage, video) && statistic
```

Comment:
Does this if statement need to be outside the `td`? I didn't check this out by running the code so I'm not sure, but there may be misalignment between table header and table body for non-admins.